tests/drivers/spi/spi_controller_peripheral/testcase.yaml

```diff
@@ -64,6 +64,16 @@ tests:
       - nrf54h20dk/nrf54h20/cpurad
       - nrf54h20dk/nrf54h20/cpuppr
 
+  drivers.spi.spis_fast:
+    # SPIS120 instance occupies P6 which is not available on nRF54H20 DK pin headers
```

Comment:
Maybe extend the comment with information that configuration is added only for compilation check. I know that there is` `build_only:true` but it might not be obvious that it cannot be run.